openlibrary/plugins/worksearch/code.py

```diff
@@ -310,8 +310,11 @@ def parse_json(raw_file):
     return json_result
 
 
-def run_solr_query(param = {}, rows=100, page=1, sort=None, spellcheck_count=None,
+def run_solr_query(param=None, rows=100, page=1, sort=None, spellcheck_count=None,
                    offset=None, fields=None, facet=True):
+    if param is None:
+        param = {}
+
```

Comment:
```suggestion
    param = param or {}
```
In this instance, other falsey values would be just as dangerous as None.